addons/website_forum/tests/test_forum_post.py

```diff
@@ -0,0 +1,29 @@
+from odoo.addons.website_forum.tests.common import TestForumCommon
+
+
+class TestForumPost(TestForumCommon):
+
+    def test_get_related_posts(self):
+        """Test the method returns 5 related posts based on tag similarity"""
```

Comment:
```python
"""Test the method returns 5 related posts based on tag similarity."""
```

you can use that flake8 plugin to lint you docrtring: https://pypi.org/project/pydocstyle/